lexicons/app/bsky/unspecced/searchActorsSkeleton.json

```diff
@@ -0,0 +1,56 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.searchActorsSkeleton",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Backend Actors (profile) search, returning only skeleton",
+      "parameters": {
+        "type": "params",
+        "required": ["q"],
+        "properties": {
+          "q": {
+            "type": "string",
+            "description": "search query string; syntax, phrase, boolean, and faceting is unspecified, but Lucene query syntax is recommended. For typeahead search, only simple term match is supported, not full syntax"
+          },
+          "typeahead": {
```

Comment:
would it make sense to use a non-normative parameter for this? something like `simple` or `quick`?

Reply:
hmmm, I feel like typeahead is pretty term-of-art for this query type (not just the term we are using). different sub-set of fields are queried, and only really works for prefix of a single token or two, character-by-character.

"simple" or "quick" would be confusing to me, i'd assume that they would return un-hydrated responses or something (responses are always hydrated).

Reply:
cool cool - i'm game for "typeahead" 👍